src/dmd/mars.d

```diff
@@ -1744,7 +1744,18 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
         {
             static if (TARGET.Linux || TARGET.OSX || TARGET.FreeBSD || TARGET.OpenBSD || TARGET.Solaris || TARGET.DragonFlyBSD)
             {
-                params.pic = 1;
+                params.pic = PIC.pic;
+            }
+            else
+            {
+                goto Lerror;
+            }
+        }
+        else if (arg == "-fPIE")
+        {
+            static if (TARGET.Linux || TARGET.OSX || TARGET.FreeBSD || TARGET.OpenBSD || TARGET.Solaris || TARGET.DragonFlyBSD)
```

Comment:
I don’t think there’s a concept of PIE on macOS. Everything is PIC.

Reply:
We'll see.